### PR TITLE
Add button search borrows

### PR DESCRIPTION
**Pull request by at-duongtruong**
## Description
A few sentences describing the overall goals of the pull request's commits.
## Reference Requirements
- Product Backlog ID: PBI-4
- Task ID: #4.1
## Migrations
NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

### CODE REVIEW

File: resources/views/backend/borrows/index.blade.php [hunk 1]
```diff
@@ -21,6 +21,25 @@
...
+                    </div>
+                    <div class="col-xs-2">
+                      <button type="submit" id="btn-search" class="btn btn-primary btn-flat">{{ __('borrows.search') }}</button>
+                    </div>
+                </form>
+              </div>
+            </div>
             <!-- /.box-header -->
             <div class="box-body">
               <table id="example2" class="table table-bordered table-hover">
```
Comment (at-trampham) [inline resources/views/backend/borrows/index.blade.php]: LGTM

File: resources/lang/en/borrows.php [hunk 2]
```diff
@@ -10,4 +10,9 @@
     'books_borrowing' => 'Book Borrowing',
     'from_date' => 'From Date',
     'end_date' => 'End Date',
+    'search' => 'Search',
+    'list_search' =>[
+        \App\Model\Borrow::TYPE_NAME => 'Name',
+        \App\Model\Borrow::TYPE_BORROWS => 'Book Borrowing',
```
Comment (at-tuanphan) [inline resources/lang/en/borrows.php]: TYPE_BORROWS => 'Book Borrowing', sao thấy nó lạ lạ vậy em ơi, 
TYPE_BORROWING => 'Book Borrowing'

**Review by ghost — COMMENTED** [review]
please check my comments

File: app/Model/Borrow.php [hunk 3]
```diff
@@ -12,7 +12,8 @@ class Borrow extends Model
      * @type int
      */
     const BORROWING = 0;
-
+    const TYPE_NAME = 'name';
+    const TYPE_BORROWING = 'borrows';
```
Comment (at-tuanphan) [inline app/Model/Borrow.php]: borrowing chứ em ???

**Review by ghost — COMMENTED** [review]
please check my comment

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**